include/flatbuffers/idl.h

```diff
@@ -406,7 +410,7 @@ struct IDLOptions {
       generate_name_strings(false),
       escape_proto_identifiers(false),
       generate_object_based_api(false),
-      cpp_object_api_pointer_type("std::unique_ptr"),
+      cpp_object_api_pointer_type("flatbuffers::unique_ptr"),
```

Comment:
Can't do this as some users of stlport (we have some internal users) have defined std::unique_ptr as well so it's not safe to just inject the type into the std namespace.  flatbuffers::unique_ptr is an alias of std::unique_ptr in all implementations except for stlport and <= MSVC2012. 

In <= MSVC2012 the derived class is a *little* tricky as this results in the user doing...

flatbuffers::unique_ptr fbp;
std::unique_ptr sup(fbp.release());

which I concede is inconvenient.  AFAIK this is the closest to type aliasing that's possible with <= MSVC2012.

BTW It's most definitely not 99% of the world not caring about stlport I'm afraid.  I've pinged you the data.

I guess another way to look at this is why support MSVC2010 / MSVC2012?